### PR TITLE
Update taxonomy map

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py [hunk 1]
```diff
@@ -197,6 +207,7 @@
                     "focus-firefox",
                     "thunderbird",
                     "thunderbird-android",
+                    "firefox-enterprise",
```
Comment (emilghittasv) [inline kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py:210]: `Relay` also needs to be added to the `Crashing and slow performance` topic product list.

File: kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py [hunk 2]
```diff
@@ -265,13 +282,13 @@
                 "title": "Downloads",
                 "slug": "downloads",
                 "description": "Manage and troubleshoot your downloads.",
-                "products": ["firefox", "mobile", "ios", "pocket"],
+                "products": ["firefox", "mobile", "ios", "pocket", "mdn-plus"],
```
Comment (emilghittasv) [inline kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py:285]: `pocket` needs to be removed from the `Downloads` product list.

File: kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py [hunk 3]
```diff
@@ -424,7 +437,6 @@
                     "firefox-enterprise",
                     "thunderbird",
                     "thunderbird-android",
-                    "mozilla-account",
```
Comment (emilghittasv) [inline kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py:427]: We also need to add `MDN Plus` inside the `Customization` topic product list.

**Review by emilghittasv — CHANGES_REQUESTED** [review]

**Comment by akatsoulas** [comment]
@emilghittasv PR udpated

**Review by emilghittasv — APPROVED** [review]

**Pull request merged**